tasks.py

```diff
@@ -8,9 +8,12 @@
 
 To show task help page `invoke <NAME> --help`
 """
+import json
+
 import invoke
 
 from scripts import check_type_hint_coverage
+from tests.integration.usage_statistics import usage_stats_utils
```

Comment:
IMO this should be in something like `/scripts` not tests, like the `check_type_hint_coverage` example above.
Shouldn't need to install `pytest` in order to run our type-checkings tasks.